image/templates/helm/stackrox-secured-cluster/templates/sensor-netpol.yaml

```diff
@@ -26,6 +26,11 @@ spec:
       - podSelector:
           matchLabels:
             app: scanner
+{{ if ._rox.scannerV4._indexerEnabled }}
```

Comment:
Please use

```
{{- if ... }}
...
{{- end ... }}
```
and while you are at it also adjust in l.25 and l.34 accordingly -- that one was overlooked it seems.

Background: when constructing YAML the semantics of whitespace eliminiation of `{{- ... -}}, {{- ... }}, {{ ... -}} and {{ ... }}` are sometimes somewhat tricky to get right. We gathered some experience regarding this when writing these charts initially. Bottom line has been: For this kind of *line-based control-flow* it seems to be always doing the right thing (what you would expect) when using `{{-` on the LHS and `}}` on the RHS.